modules/migration/components/MigrationBanner.tsx

```diff
@@ -21,19 +21,25 @@ export function MigrationBanner(): React.ReactElement | null {
     isDelegatedToExpiredContract,
     isDelegateContractExpired,
     isDelegateContractExpiring,
-    isShadowDelegate
+    isShadowDelegate,
+    isDelegateContractV1,
+    isDelegatedToV1Contract
```

Comment:
Minor thing but we make name consistent here, `V1Contract`. They are currently flipped around in each variable name.

Reply:
good catch